extension/src/experiments/columns/collect/metricsAndParams.ts

```diff
@@ -8,22 +8,20 @@ import {
 } from './util'
 import { ColumnType } from '../../webview/contract'
 import {
-  ExperimentFields,
+  ExpData,
   isValueTree,
   Value,
   ValueTree,
-  ValueTreeOrError,
-  ValueTreeRoot
+  FileDataOrError,
+  fileHasError,
+  MetricsOrParams
 } from '../../../cli/dvc/contract'
 import {
   buildMetricOrParamPath,
   FILE_SEPARATOR,
   METRIC_PARAM_SEPARATOR
 } from '../paths'
 
-export const typedValueTreeEntries = (value: NonNullable<ValueTree>) =>
```

Comment:
[F] No longer need this now the types have been fixed... or TypeScript can handle typing object entries now.